.github/workflows/bump_commit_release_branch.yml

```
@@ -59,7 +59,7 @@ jobs:
           echo "New version (bump_version): $new_version"
 
           # create PR
-          git push --quiet --set-upstream origin workflow/create-bump-commit-release-branch-${{ inputs.releaseBranchVersion }}
+          git push --quiet --set-upstream origin workflow-${{ github.run_id }}/create-bump-commit-release-branch-${{ inputs.releaseBranchVersion }}
```

folder structure like `workflow/`, `feature`, ... can help organizing the branches. So I would do something like this:
```suggestion
          git push --quiet --set-upstream origin workflow/create-bump-commit-release-branch-${{ inputs.releaseBranchVersion }}-${{ github.run_id }}
```